getting-started/what-you-need.md

```diff
@@ -1,102 +1,118 @@
 ---
-title: What You Need
-page_title: What You Need to Use the Telerik Blazor Components
-description: The packages and assets you need to use the Telerik UI for Blazor components, how to get them, and how to configure your project to include the Telerik Blazor components.
+title: Typical Workflow
```

Comment:
I personally do not find this suggestive of what the content will be. Do we have feedback prompting this change?

Reply:
This change is based on my observations. The main issue is that the title "What You Need" isn't grammatically consistent with the rest of the titles in the parent section. Furthermore, the title isn't informative enough because it doesn't define a specific purpose&mdash;installing, configuring, using, or something else.
"Typical Workflow", on the other hand, has a similar grammatical structure and implies a complete process or a sequence of steps needed to complete a task&mdash;exactly what we have in the article.